packages/next/src/server/config.ts

```diff
@@ -300,6 +324,12 @@ function assignDefaults(
     }
   }
 
```

Comment:
Maybe let's add a comment to remove this after the next minor

Reply:
👍 add a TODO comment